tests/test_test_object_accessed.py

```diff
@@ -0,0 +1,73 @@
+import os
+import unittest
+
+from os.path import exists
+from unittest.mock import patch
+
+import helper
+
+class TestTestObjectAccessed(unittest.TestCase):
+
+  def setUp(self):
+    self.data = {
+      "DC_PEC": '',
+      "DC_CODE": '''
+import numpy as np
+arr = np.array([1, 2, 3])
+x = arr.shape
+print(arr.data)
+      ''',
+      "DC_SOLUTION": '''
+      # not used
+      '''      
+    }
+
+  def test_objectOnly(self):
+    self.data["DC_SCT"] = 'test_object_accessed("arr")'
+    sct_payload = helper.run(self.data)
+    self.assertEqual(sct_payload['correct'], True)
+
+    self.data["DC_SCT"] = 'test_object_accessed("arr", times=2)'
+    sct_payload = helper.run(self.data)
+    self.assertEqual(sct_payload['correct'], True)
+
+    self.data["DC_SCT"] = 'test_object_accessed("arr", times=3)'
+    sct_payload = helper.run(self.data)
+    self.assertEqual(sct_payload['correct'], False)
+    self.assertEqual(sct_payload['message'], "Have you accessed <code>arr</code> at least 3 times?")
+
+    self.data["DC_SCT"] = 'test_object_accessed("arr", times=3, not_accessed_msg="silly")'
+    sct_payload = helper.run(self.data)
+    self.assertEqual(sct_payload['correct'], False)
+    self.assertEqual(sct_payload['message'], "silly")
```

Comment:
The reason you do tests in separate methods is that it is more granular if something goes wrong (faster to find the bug). You know exactly where the problem is. Doesn't have to be pretty. I like that you use 1 SCT each time, but I'd make each test one method next time.